src/components/organisms/TaskCreator/TaskCreator.tsx

```diff
@@ -38,22 +34,22 @@ const TaskCreator = () => {
 
   if (!isLoading) {
     return (
-      <TaskCreatorContainer>
+      <S.TaskCreatorContainer>
         <Button onClick={handleAddItem} text="add"></Button>
-        <TaskCreatorInput
+        <S.TaskCreatorInput
           type="text"
           placeholder="Create a new todo..."
           maxLength={200}
           value={text}
           onChange={handleTextChange}
         />
-      </TaskCreatorContainer>
+      </S.TaskCreatorContainer>
     );
   } else {
     return (
-      <BusyAnimation>
+      <S.BusyAnimation>
         <BusyIcon className="busy-icon" />
-      </BusyAnimation>
+      </S.BusyAnimation>
     );
   }
 };
```

Comment:
zrób ternary zamiast else if

Reply:
zrobiłam else if bo ternary było mi ciężko zobaczyć gdzie się zaczyna i gdzie kończy przez to że to jest długie

Reply:
to lepiej zrobić dwa osobne komponenty, przekazywać propsy i wtedy zrobić isLoading ? <Comp1 "jakieś propsy"> : <Comp2 "jakieś propsy">